src/spikeinterface/core/recording_tools.py

```diff
@@ -910,3 +910,35 @@ def get_rec_attributes(recording):
         dtype=recording.get_dtype(),
     )
     return rec_attributes
+
+
+def check_recording_attributes_match(recording1, recording2_attributes, skip_properties=True):
```

Comment:
```suggestion
def check_recording_attributes_match(recording1, recording2_attributes, skip_properties=True) -> bool:
```
For a function like this it isn't clear to me that this returns a bool. So I prefer knowing that this just returns a bool. I say this because there are a couple checks in Neo that return Nothing to indicate passing and raise an error to indicate a problem. This returns a bool to say everything is okay. You have it in the docstring, but also seeing it in the type hint is nice (I know we are inconsistent with type hints so we can ignore this if we don't want the type hint).